### ISSUE TITLE
jsx-a11y plugin does not allow to run "npm run build" command

### ISSUE THREAD

**Issue by rickystar04** [rickystar04/PicPortal#2]
>npm run build
npm WARN config global `--global`, `--local` are deprecated. Use `--location=global` instead.

> my-app@0.1.0 build
> react-app-rewired build

Creating an optimized production build...
Failed to compile.

[eslint] Failed to load plugin 'jsx-a11y' declared in 'package.json » eslint-config-react-app': ..\PicPortal\node_modules\language-subtag-registry\data\json\registry.json: Unexpected end of JSON input
